pkgs/os-specific/linux/kernel/xanmod-kernels.nix

```diff
@@ -7,6 +7,12 @@ let
     hash = "sha256-MeH9RUPDiuN22eAZ18v+N3aIT18dQ3FnTkcQV0MjB4k=";
   };
 
+  stableTTVariant = {
```

Comment:
```suggestion
  devVariant = {
```

Reply:
It isn't guaranteed that it will be TT forever. So I'm leaning towards naming it `dev` instead since it's under development.